langgenius/dify
# Added OceanBase as an option for the vector store in Dify

# Checklist:

> [!IMPORTANT]  
> Please review the checklist below before submitting your pull request.


# Description

This PR added [OceanBase CE](https://github.com/oceanbase/oceanbase) as an option for the vector store in Dify.

Fixes #9982.

## Type of Change


# Testing Instructions

I have set `VECTOR_STORE=oceanbase` to enable oceanbase to act as the vector store of dify and tested functionalities of datasets. Creating dataset, uploading documents, retrieving chunks and deleting dataset work as expected. 

The followings are screenshots of the test,

<img width="1837" alt="image" src="https://github.com/user-attachments/assets/630368a9-e84f-4029-aa22-30034f1aee2c">

<img width="2183" alt="Pasted Graphic 1" src="https://github.com/user-attachments/assets/93fd7695-b5fe-4d76-90bf-2fe589881c77">
